Testing/Test.java

```diff
@@ -69,10 +78,10 @@ private String getCorrectOutput(String filename){
             System.out.println("File not found: "+filename);
         }
         Scanner s = new Scanner(r);
-        String correctOutput = "";
-        while(s.hasNext()) correctOutput += s.nextLine();
+        StringBuilder correctOutput = new StringBuilder();
```

Comment:
With a StringBuilder object you don't have to create a new StringBuilder for every concatenation.